megamek/unittests/megamek/common/autoresolve/damage/DamageApplierChooserTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2025 The MegaMek Team. All Rights Reserved.
+ * Copyright (C) 2025 The MegaMek Team. All Rights Reserved.
  *
  * This file is part of MegaMek.
```

Comment:
Don't remove a starting year on a copyright unless the file and code didn't exist before said year.

Reply:
the starting year was put in there a a bad copy-paste, this file was created in 2024, not 2017, I did however failed to fix the year to 2024